### DISCUSSION TITLE
Best way to generate initial ids?

### DISCUSSION THREAD

**Discussion by matthew-dean** [ai/nanoid#432] [General]
In HTML / CSS, identifiers can be alphanumeric but MUST start with `[A-Za-z]`. (Well, CSS identifiers can also have a dash prefix, but for the sake of argument, lets say this is aligned.

I realize I can pass an alpha dictionary to nanoid to only use alpha characters, but could you add a feature of easily specifying this format, so that I can generate valid IDs for `<label for={id}>` `<input id={id}>` pairings?

**Comment by ai** [comment]
Nano ID is bad option for that simple case. You will use too much hardware resources for that.

Just use `useId()` or small snippet like `"id" + (++counter)`

**Reply by matthew-dean** [reply]
Okay thanks for the suggestion
